tests/fixtures/commands/drupal-services-setup.yml

```diff
@@ -0,0 +1,12 @@
+- config: |
+    my_settings:
+      foo: bar
+    drupal:
+      service_parameters:
+        settings:
+          foo: ${my_settings.foo}
+  expected:
+    - contains: |
+        parameters:
+          settings:
+            foo: bar
```

Comment:
We should also test:
1. The override option
2. Empty parameters

Reply:
Done.